CREDITS.txt

```diff
@@ -2,45 +2,48 @@ Minecraft Forge: Credits/Thank You
 
 Forge is a set of tools and modifications to the Minecraft base game code to assist 
 mod developers in creating new and exciting content. It has been in development for 
-several years now, but I would like to take this time thank a few people who have 
-helped it along it's way.
+several years now, so now I would like to take this time thank a few people who have 
+helped it along its way.
 
-First, the people who originally created the Forge projects way back in Minecraft 
-alpha. Eloraam of RedPower, and SpaceToad of Buildcraft, without their acceptiance 
-of me taking over the project, who knows what Minecraft modding would be today.
+First, thanks to the people who originally created the Forge projects way back in
```

Comment:
No need for `to` before the names of people because you are enumerating `a few people`.